python/cugraph/cugraph/structure/renumber_wrapper.pyx

```diff
@@ -49,14 +49,20 @@ cdef renumber_helper(shuffled_vertices_t* ptr_maj_min_w, vertex_t, weights):
         move(pair_s_minor.first), vertex_t, "shuffled_minor")
 
```

Comment:
Can you do this instead (and also for `shuffled_weight_series` below)? This might be more self-documenting, and should also address @BradReesWork 's concern about always doing two operations on the df.
```suggestion
    if shuffled_major_series is None:
        shuffled_major_series = cudf.Series(dtype=vertex_t)
    if shuffled_minor_series is None:
        shuffled_minor_series = cudf.Series(dtype=vertex_t)
```

Reply:
Addressed